javascript/modules/client/src/context.ts

```diff
@@ -6,7 +6,7 @@ import { UnsupportedProtocolError } from "@aragon/sdk-common";
 export { ContextParams } from "./internal/interfaces/context";
 
 const supportedProtocols = ["https:"];
-if(process.env.CI) {
+if(process.env.TESTING) {
```

Comment:
I see this was already introduced, but I'm not sure if the NodeJS-only `process.env` will bring us trouble when running on the browser